autoload/neomake/statusline.vim

```diff
@@ -184,9 +186,13 @@ function! neomake#statusline#get_status(bufnr, options) abort
     if !empty(running_jobs)
         let format_running = get(a:options, 'format_running', '… ({{running_job_names}})')
         if format_running isnot 0
-            return s:formatter.format(format_running, {
-                        \ 'bufnr': a:bufnr,
-                        \ 'running_jobs': running_jobs})
+            let args = {'bufnr': a:bufnr, 'running_jobs': running_jobs}
+            for l:opt in ['running_jobs_separator', 'running_jobs_project_marker']
```

Comment:
No need to use `l:` prefixes.

Reply:
Instead of `running_jobs_project_marker` we could have `format_running_job_project` and `format_running_job_file`.